### ISSUE TITLE
Orphaned Fingerprint Labels on Change Detection when a Schema is deleted. 

### ISSUE THREAD

**Issue by whitingjr** [Hyperfoil/Horreum#1663]
**Describe the bug**

When deleting a Schema the Fingerprint Labels still remain on Change Detection.

![orphaned-fingerprint-labels](https://github.com/Hyperfoil/Horreum/assets/708428/0cc81e0d-4353-4b4c-9ca9-50084e20567b)

**To Reproduce**

Create a Schema
Create a Label
Create a Test
Add the created Label as a Fingerprint Label
Delete the Schema containing the Label
Reload the Test and check the Change Detection tap page and the Label is still there.

The ideal situation is that when the Label is deleted along with the Schema the DELETE cascades to the table that stores the reference.

**Version**

```
0.12.2
```

**Java**

```
17
```

**Comment by johnaohara** [comment]
On Hold due to: https://github.com/Hyperfoil/Horreum/discussions/1603

**Comment by whitingjr** [comment]
Importing a Schema with this Label definition contains an access property with the value of "PUBLIC".

	```
"labels" : [ {
    "name" : "kb_report_results_podLatencyQuantilesMeasurement_quantiles_Ready_P99",
    "filtering" : true,
    "metrics" : true,
    "schemaId" : "221",
    "acccess" : "PUBLIC",
    "owner" : "ocp-perfscale-team",
    "extractors" : [ {
      "name" : "P99",
      "path" : "$.results.podLatencyQuantilesMeasurement.quantiles.Ready.P99",
      "isarray" : false
    } ]
  }
```
The unmarshalling of the Label to an object is not correctly creating the object `access` property correctly with a value. As can be seen in the SchemaService.importSchema method inspecting the Schema object before persisting.
![import-schema-labels-empty-access](https://github.com/Hyperfoil/Horreum/assets/708428/8f5fc49f-b3ff-4a99-b7a4-b2a31b02b435)

**Comment by johnaohara** [comment]
I don't think we should spend time investigating this if we are going to remove schemas
